src/dotnet/commands/dotnet-new3/Installer.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.TemplateEngine.Edge;
+using Microsoft.TemplateEngine.Edge.Settings;
+using Microsoft.TemplateEngine.Utils;
+
+namespace Microsoft.DotNet.Tools.New3
+{
+    internal class Installer : IInstaller
+    {
+        public void InstallPackages(IEnumerable<string> installationRequests)
+        {
+            List<string> localSources = new List<string>();
```

Comment:
@jonsequitur how strongly do you feel on this? These files come from here https://github.com/dotnet/templating/tree/master/src/dotnet-new3 where we have different code style guidelines & would prefer not to have to switch explicit types in cases where they can be easily inferred (like immediate assignment from new) when rolling from a "preview" back in to the what will be `dotnet new` by the time this PR is ready to merge.